distribution/changelog.txt

```diff
@@ -23,6 +23,7 @@
 - Fix: [#15199] Construction window is not closed when a ride gets demolished.
 - Fix: [#15255] Tile Inspector shows banner information on walls that do not contain one.
 - Fix: [#15289] Unexpected behavior with duplicated banners which also caused desyncs in multiplayer.
+- Fix: [#15257] Chat icon shows in scenario/track editor. Other icons don't disable when deactivated in options menu.
```

Comment:
Please move this up one line so they are ordered by number.

Reply:
Whoops. Fixed that.